package.json

```diff
@@ -71,6 +71,7 @@
   },
   "dependencies": {
     "@suitest/translate": "^2.0.5",
+    "@types/node": "^13.9.3",
```

Comment:
Shouldn't this be in dev dependencies?

Reply:
We use it for describing Buffer in .d.ts files - that why I put it to "dependecies"